{"name": "test5", "status": "passed", "description": "Test 5: real input MS, MS has several SPWs observed in parallel - not supported, expected error", "attachments": [{"name": "stderr", "source": "a2ede89d-4afe-4d63-aa39-db2aea1534ab-attachment.txt", "type": "text/plain"}], "start": 1758034744565, "stop": 1758034744758, "uuid": "e47fcd03-6192-4b10-98c3-5207b9e1d7b3", "historyId": "0ddc090aed67c87a7e162876800e508f", "testCaseId": "0ddc090aed67c87a7e162876800e508f", "fullName": "casatools.test_tool_sdm.SdmFrommsTest#test5", "labels": [{"name": "parentSuite", "value": "casatools"}, {"name": "suite", "value": "test_tool_sdm"}, {"name": "subSuite", "value": "SdmFrommsTest"}, {"name": "host", "value": "casa-rockylinux8-amd64-perf07"}, {"name": "thread", "value": "593221-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "casatools.test_tool_sdm"}], "titlePath": ["casatools", "test_tool_sdm.py", "SdmFrommsTest"]}